This file is part of MXE.
See index.html for further information.

From 088fa6af785209f6b6230dcba5b26583a68be6fc Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sun, 6 Sep 2015 22:53:33 +0100
Subject: [PATCH 2/5] fix extra qualification 'Handshake::'

Compilation error.
---
 libmysql/authentication_win/handshake.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmysql/authentication_win/handshake.h b/libmysql/authentication_win/handshake.h
index 14b1386..395af36 100644
--- a/libmysql/authentication_win/handshake.h
+++ b/libmysql/authentication_win/handshake.h
@@ -100,7 +100,7 @@ public:
   Handshake(const char *ssp, side_t side);
   virtual ~Handshake();
 
-  int Handshake::packet_processing_loop();
+  int packet_processing_loop();
 
   bool virtual is_complete() const
   {
-- 
2.1.4

